E2E/assertions/assertions.ts

```diff
@@ -0,0 +1,40 @@
+const { I, game } = inject();
+
+export = {
+    async checkIfTheLinkWasCopied (currentUrl) {
+      I.usePlaywrightTo('check clipboard', async ({ page, browserContext }) => {
+        await browserContext.grantPermissions(['clipboard-read']);
+        const clipboardText = await page.evaluate("navigator.clipboard.readText()");
+        I.assertEqual(currentUrl, clipboardText);
+      });
+  },
+
+  async checkIfTheSPIsRoundedCorrectlyForTheFirstPlayer() {
+    const averageFirstPlayerSP = game.calculateAverage("bishop", "d", 3);
+    let expectedFirstPlayerScore = game.checkPlayerSP(averageFirstPlayerSP);
```

Comment:
`checkPlayerSP` should have a better name, it tells nothing about what the function actually does